src/online2bin/extend-wav-with-silence.cc

```diff
@@ -76,16 +76,47 @@ int main(int argc, char *argv[]) {
       exit(1);
     }
 
-    std::string wav_rspecifier = po.GetArg(1);
-    std::string wav_wspecifier = po.GetArg(2);
+    if (ClassifyRspecifier(po.GetArg(1), NULL, NULL) != kNoRspecifier) {
```

Comment:
It's not very clear to me what you're doing here since it's so large. Since it looks like you made a new set of input options, can you change the usage string of this binary?

Reply:
I am adding support for `rxfilename` and `wxfilename` so it can be used in a piped command.
I'll update the usage.

Reply:
Great, thanks!